orchestrator/index.js

```diff
@@ -0,0 +1,67 @@
+const request = require('request-promise');
```

Comment:
We haven't decided yet if we need to use semicolons or not, to keep the code format until that point, please remove your semicolons and run eslint: `npm run lint`.